tests/Functional/app/ORM/config.yml

```diff
@@ -10,6 +10,8 @@ doctrine:
         auto_mapping: false
 
 services:
+    _defaults:
+        public: true
```

Comment:
That should only apply to service declared here, that's why it doesn't work ;)

Reply:
Can you add 
```yml
fos_elastica.manager.orm:
    public: true
```

Reply:
I'm pretty sure this will fix the error ;)

Reply:
Perhaps this might work as well:

```yaml
services:
    _defaults:
        public: true
    test_alias.fos_elastica.indexable:
        alias: fos_elastica.indexable
    indexable_service:
        class: FOS\ElasticaBundle\Tests\Functional\app\ORM\IndexableService
    fos_elastica.manager.orm:
        class: FOS\ElasticaBundle\Doctrine\RepositoryManager
```

Reply:
Yes because these service declared here will be public, so this should work as well ;)